Gopkg.toml

```diff
@@ -52,3 +52,11 @@
 [prune]
   go-tests = true
   unused-packages = true
+
+[[constraint]]
+  name = "cloud.google.com/go"
+  version = "0.26.0"
+
+[[constraint]]
+  branch = "master"
+  name = "google.golang.org/api"
```

Comment:
Hi @josh-padnick, I agree that the new Google Cloud Platform Go SDK looks like the right way to go, but for reasons, I'm not entirely sure of, it seems they are still yet to implement all of the necessary API calls. For example, there are essentially no available methods for working with Compute Instances. I noticed in their own integration tests they also use the autogenerated library (see: https://github.com/GoogleCloudPlatform/google-cloud-go/blob/8007d2620d6fbc9f7eeb0a0ee571fe802b866353/storage/go110.go).

In addition I found other tools like Terraform also depend on this SDK (see: https://github.com/terraform-providers/terraform-provider-google/blob/master/google/resource_storage_bucket_object.go#L14).

I had to use `branch = "master"` because it appears they don't push tags or use the GitHub release system which is required for Dep. Maybe I'm wrong?